src/dawn/engine_dawn.cc

```diff
@@ -334,6 +335,77 @@ Result EngineDawn::DoClearDepth(const ClearDepthCommand* command) {
   return {};
 }
 
+uint32_t Align(uint32_t value, size_t alignment) {
```

Comment:
Needs a method comment.   
In particular, alignment must be a power of 2.  Otherwise the math doesn't work.